index.html

```diff
@@ -58,5 +58,81 @@ <h2>Software Developer</h2>
             </ul>     
         </div>
     </section>
+    <section id="work">
+        <div class="work">
+            <img src="images/arrow.svg" alt="arrow">
+            <h2>work</h2>
+        </div>
+        <div class="recent-work">
+
+            <h1>MY RECENT WORK</h1>
+            <hr>
+        </div>
+        <div class="background1 background">
+            <div class="stories">
+                <img src="images/placeholder.png" alt="placeholder">
+            </div>
+            <div class="holder">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="list">
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li class="ruby">Ruby</li>
+                </ul>
+                <button type="button" class="btn">See Project</button>
+            </div>
+        </div>
+        <div class="background2 background">
+            <div class="stories1">
+                <img src="images/placeholder.png" alt="placeholder">
+            </div>
+            <div class="holder">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="list">
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li class="ruby">Ruby</li>
```

Comment:
- Please use the appropriate font size for the paragraph as it is in the Figma design `inter` 
- Please use the proper color of the divider in between each technology used for the project, yours is currently white, use the Figma color

##### Your design 
![Screenshot from 2022-06-30 08-04-04](https://user-images.githubusercontent.com/58771507/176614048-dee2403c-49b1-4d6c-8b6c-28152b415c96.png)

##### Figma design

![Screenshot from 2022-06-30 08-04-25](https://user-images.githubusercontent.com/58771507/176614072-f9caf21e-1791-4283-a8fa-125e96f9c4ac.png)